typescript-and-json/main.ts

```diff
@@ -0,0 +1,24 @@
+const array: { isbn: string; title: string; author: string }[] = [
+  {
+    isbn: '23344500576',
+    title: 'Tomorrow and Tomorrow...',
+    author: 'Gabrielle LastName',
+  },
+  { isbn: '74747474747474', title: 'Steve Jobs', author: 'Walter Isaacson' },
+  { isbn: '123434322', title: 'Elon Musk', author: 'Walter Isaacson' },
+];
+
+console.log('Array of books: ', array);
+console.log('type of array: ', typeof array);
+
+const booksJSON = JSON.stringify(array);
+console.log('booksJSON: ', booksJSON);
+console.log('type of booksJSON: ', typeof booksJSON);
+
+const studentJSON = '[{"Number id":2302, "String name":"Nader Motamedi"}]';
```

Comment:
Couple things!

1. No array, just an object
1. Properties shouldn't have spaces in them. The property names should just be `id` & `name` respectively

Reply:
I see!